src/commonMain/kotlin/fr/acinq/lightning/channel/Syncing.kt

```diff
@@ -49,7 +46,13 @@ data class Syncing(val state: PersistedChannelState, val waitForTheirReestablish
                         state
                     }
                     val channelReestablish = nextState.run { createChannelReestablish() }
-                    val actions = listOf(ChannelAction.Message.Send(channelReestablish))
+                    val actions = buildList {
+                        if (nextState != state) {
+                            // we just restored from backup
+                            add(ChannelAction.Storage.StoreState(nextState))
```

Comment:
We can probably update one of our restore unit tests to check that `StoreState` is emitted here?

Reply:
Clearly, I'm in-between rebases and this was meant as a reminder. Let put it back to Draft.

Reply:
Done in [8b37758](https://github.com/ACINQ/lightning-kmp/pull/429/commits/8b377580ebbdf362b2a285b63be7a172d2267668).